collaboration.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Collaboration Tools
+layout: topic
+categories: guide
+---
+A key component of a successful hackathon is being able to efficiently 
+collaborate with other members of your team. There are many different 
+free tools that can help you with this. Some collaboration tools that 
+might support your project include but are not limited to:
+
+## Code Collaboration Tools
+- [GitHub][github-home]
+- [BitBucket][bitbucket]
+
+## Project Management Tools
+- [GitHub Projects][gh-projects]
+- [Trello][trello]
+
+## Documentation Collaboration Tools
+- [Google Drive][gdrive]
+
+# GitHub
+
+For many developers, Git is an invaluable tool of any software
+development stack. A version control system is a great way
+to keep your projects backed up, centralized, tracked, and
+accessible to be worked on with others.
+
+GitHub is a website that builds on the functionality of Git,
+it hosts your projects in a single place and adds some useful
+project tracking features that can be accessed in its easy-to-use
+website. You can [make an account here][github-join] and
+[sign up for the student discounts here.][gh-student-pack]
+
+Git is (normally) just a console application, but many IDEs
+have built in support for git version control tracking.
+  - [Git for Visual Studio][git-for-vs]
```

Comment:
For parallelism, maybe rephrase to "Git integration for Visual Studio" (like the JetBrains line below)